src/planning/include/planning/Planner.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
This file is obsolete and can be removed.

Reply:
Removed.